# Hedef Takip Bilgileri
Header header

# Hedef tespit durumu
bool kitlenme

# Hedef uçağın bilgileri
int32 takim_id
float64 enlem
float64 boylam
float64 irtifa

# Takip parametreleri
float32 takip_mesafesi  # metre (varsayılan: 10m)
bool takip_aktif        # Takip aktif/pasif
bool acil_durdurma      # Acil durdurma
